src/modules/reportes_poai/reportes_poai.service.ts

```
@@ -0,0 +1,665 @@
+        return await this.generarReportePoaiDirecto(historialId);
+      }
+
+    } catch (error) {
+      throw new Error(`Error generando reporte: ${error.message}`);
+    }
+  }
+
+  // Versión alternativa usando consulta directa (sin función almacenada)
+  async generarReportePoaiDirecto(historialId: number): Promise<Buffer> {
+    try {
+      const query = `
```

ESLint

[error] 'query' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)